ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import timestamp from 'core/utils/timestamp';
+import { getUnixTime } from 'date-fns';
+
+import type StoreService from 'vault/services/store';
+import type { ClientsRouteModel } from '../clients';
+
+export default class ClientCountTokenRoute extends Route {
+  @service declare readonly store: StoreService;
+
+  queryParams = {
+    start_time: {
+      refreshModel: true,
+    },
+    end_time: {
+      refreshModel: true,
+    },
+  };
+
+  currentTimestamp = getUnixTime(timestamp.now());
+
+  async getActivity(start_time: number, end_time: number) {
+    let activity, activityError;
+    try {
+      activity = await this.store.queryRecord('clients/activity', {
+        start_time: { timestamp: start_time },
```

Comment:
the query params are always formatted as unix now, right? Like when a user selects a date we immediately convert to unix? Originally this object was added because the date was formatted differently depending on how it was set. We can just pass these query params directly, if they're always unix time now and then remove the extra logic (`formatQueryParams`) in the activity adapter

Though the comment explaining that javascript localizes Date() may be good to keep above the `queryRecord` method so we don't repeat our past mistake of sending the query with a timezone

Reply:
That's something to keep in mind once we fully move over the `token` logic and delete the original route but for now it will be necessary so we can cross reference things as we continue to make changes. I agree though that we should go over things and simplify and remove unnecessary left overs of the original implementation.

Reply:
Ahh good point - I forgot we've left `token.js` alone _mostly_ for now